Script/Packages/Atomic/UI.json

```diff
@@ -22,7 +22,9 @@
 		    "getWidgetAt<T extends UIWidget>(x: number, y: number, include_children: boolean): T;"
 		],
 		"UIWidget": [
-		    "getWidget<T extends UIWidget>(id: string): T;"
+		    "getWidget<T extends UIWidget>(id: string): T;",
+		    "onEvent: () => void;",
```

Comment:
Good getting these added to the TS defs, ```onEvent``` takes an Atomic.UIWidgetEvent parameter: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/AtomicJS/Javascript/JSUI.cpp#L273, you can find the TypeScript decl in the generated Atomic.d.ts file for reference

```typescript
** object returned in the callback for the WidgetEvent event.**/
    export interface UIWidgetEvent extends Atomic.EventData {
        /**  pointer of widget's OnEvent we are in */
        handler : Atomic.UIWidget;
        target : Atomic.UIWidget;
        type : Atomic.UI_EVENT_TYPE;
        x : number;
        y : number;
        deltaX : number;
        deltaY : number;
        count : number;
        key : number;
        specialKey : number;
        modifierKeys : number;
        refid : string;
        touch : boolean;
        /**  -> bool */
        /** Unmapped Native Type: [OUT] */
        handled : any;
    }
```